flink-end-to-end-tests/flink-stream-state-ttl-test/src/main/java/org/apache/flink/streaming/tests/verify/ValueWithTs.java

```diff
@@ -56,8 +57,8 @@ public String toString() {
 	/** Serializer for Serializer. */
 	public static class Serializer extends CompositeSerializer<ValueWithTs<?>> {
```

Comment:
This serializer class previously did not have a `serialVersionUID` defined.
Need to explicitly set it to what it was before, to be on the safe side here.

Reply:
Ah scratch that, just realized that this is only a serializer used in tests.